providers/shared/components/containerhost/id.ftl

```diff
@@ -0,0 +1,32 @@
+[#ftl]
+
+[@addComponentDeployment
+    type=CONTAINERHOST_COMPONENT_TYPE
+    defaultGroup="solution"
+/]
+
+[@addComponent
+    type=CONTAINERHOST_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type" : "Description",
+                "Value" : "An autoscaling container host cluster"
+            },
+            {
+                "Type" : "Providers",
+                "Value" : [ "aws" ]
+            },
+            {
+                "Type" : "ComponentLevel",
+                "Value" : "solution"
+            }
```

Comment:
These are no longer required and with the move to deployment groups and placements we probably don't need them anymore

Reply:
Cheers, have removed in e98df10